package.json

```diff
@@ -121,6 +121,12 @@
           "scope": "window",
           "type": "string"
         },
+        "ty.inlayHints.variableTypes": {
+          "default": false,
```

Comment:
Can you tell me more about the reason for defaulting to false?

Reply:
I'm going to change the default back to `true` to avoid any breakage, we can decide later if it should be off by default or not.